FileContentOrdering.md

```diff
@@ -62,3 +62,56 @@ internal func updateUnalliedHouses(_ houses: [House]) {
 private maximumHouseSize: Int = 50
 ```
 
+## Convention - Grouping
+Use `// MARK: Some text comment` to group related methods into sections. For example, in a view controller class, there might be `Setup`, `Layout`, and `Event Handlers` sections. 
+When marking a section of methods belonging to a superclass or a protocol, use the actual name of the superclass or protocol type instead of a loose translation. If there are multiple protocols, methods should be grouped by individual protocols.
+
+## Rationale
+Grouping related methods into sections help with code organization and navigation.
+
+## Examples
+
+``` swift
+
+// bad: does not use the name of the protocol as the name of the section
+// MARK: - Table view delegate
+...
+
+// good: use the name of the protocol directly as the name of the section
+// MARK: - UITableViewDelegate
```

Comment:
While I think this is acceptable here, I don't think the "bad" example is particularly egregious. I think this change has overloaded Xcode's picking up of extensions and the `// MARK:` syntax. With this, we are requiring duplication, which isn't usually a good thing.

As an example, see line 86 where we are saying that combining two delegates is a bad thing. Elsewhere we are saying we should use separate extensions for this. It would be good to be consistent.

Reply:
> Elsewhere we are saying we should use separate extensions for this. It would be good to be consistent.
Where are we suggesting to use separate extensions for protocol inheritance? The Delegates section? 
Yeah if this is within an extension then it's duplication, I can be more specific that this example is specific to when extensions are not used.

Reply:
Yeah that's where. I think clarification here would be good.

Reply:
Added more clarification